tests/imagebasedupgrade/tests/happy-path-upgrade.go

```diff
@@ -0,0 +1,58 @@
+package ran_du_system_test
```

Comment:
Let's maybe use a more suggestive name for the package here, e.g `ibu_happy_path`, wdyt?

Reply:
Suggested package name "tests". I will make of use it. 
Done.